README.md

```diff
@@ -123,6 +125,8 @@ For an in-depth understanding of each model's version and lifecycle, especially
 ### Google Models on Hugging Face
 - **Google flan-t5-large**: The [flan-t5-large](https://huggingface.co/google/flan-t5-large) model is accessed via Hugging Face's API.
 - **Google gemma-7b-it**: The [gemma-7b-it](https://huggingface.co/google/gemma-7b-it) model is accessed via Hugging Face's API. 
+- **Google gemma-1.1-7b-it** : The [gemma-1.1-7b-it](https://huggingface.co/google/gemma-1.1-7b-it) model is accessed via Hugging Face's API. 
+- **Google gemma-1.1-2b-it** : The [gemma-1.1-7b-it](https://huggingface.co/google/gemma-1.1-2b-it) model is accessed via Hugging Face's API.
```

Comment:
Is this accurate? "model is accessed via Hugging Face's API."
I thought you downloaded it and ran the evaluation locally.

Reply:
Thanks for pointing it out. I have updated the information.